sys-boot/lilo/lilo-24.2-r3.ebuild

```
@@ -0,0 +1,197 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit optfeature toolchain-funcs
+
+DOLILO_V="0.6"
+
+DESCRIPTION="LInux LOader, the original Linux bootloader"
+HOMEPAGE="https://www.joonet.de/lilo/"
+
+DOLILO_TAR="dolilo-${DOLILO_V}.tar.bz2"
+SRC_URI="
+	https://www.joonet.de/lilo/ftp/sources/${P}.tar.gz
+	mirror://gentoo/${DOLILO_TAR}
+"
+
+LICENSE="BSD GPL-2"
+SLOT="0"
+KEYWORDS="-* ~amd64 ~x86"
+
+IUSE="static minimal pxeserial device-mapper"
+
+DEPEND="sys-devel/bin86"
+RDEPEND="device-mapper? ( sys-fs/lvm2 )"
+
+PATCHES=(
+	"${FILESDIR}"/${PN}-24.2-Makefile-tweaks.patch
+	"${FILESDIR}"/${PN}-24.2-add-nvme-support.patch
+	"${FILESDIR}"/${PN}-24.x-fix-gcc-10.patch
+	"${FILESDIR}"/${PN}-24.x-check-for-__GLIBC__.patch
+	"${FILESDIR}"/${PN}-24.2-fix-gcc-15.patch
+	"${FILESDIR}"/${PN}-24.2-musl.patch
+)
+
+# Bootloaders should not be using arbitrary CFLAGS without good reason.  A bootloader
+# is typically only executed once to boot the system, and it should work the first time.
+QA_FLAGS_IGNORED="/sbin/lilo"
+
+src_prepare() {
+	default
+
+	# this patch is needed when booting PXE and the device you're using
+	# emulates vga console via serial console.
+	# IE..  B.B.o.o.o.o.t.t.i.i.n.n.g.g....l.l.i.i.n.n.u.u.x.x and stair stepping.
+	use pxeserial && eapply "${FILESDIR}/${PN}-24.1-novga.patch"
+}
+
+src_configure() {
+	if ! use device-mapper; then
+		sed -i make.vars -e 's|-DDEVMAPPER||g' || die
+	fi
+}
+
+src_compile() {
+	# lilo needs this. bug #140209
+	export LC_ALL=C
+
+	# we explicitly prevent the custom CFLAGS for stability reasons
+	if use static; then
+		local target=alles
+	else
+		local target=all
+	fi
+
+	emake CC="$(tc-getCC) ${LDFLAGS}" ${target}
+}
+
+src_install() {
+	emake DESTDIR="${ED}" install
+
+	if use !minimal; then
```

Nit: 
```suggestion
	if ! use minimal; then
```

Feel free to not make the change.